lambdas/session/src/test/java/uk/gov/di/ipv/cri/address/api/handler/SessionHandlerTest.java

```diff
@@ -47,18 +50,21 @@ class SessionHandlerTest {
 
     @Mock private EventProbe eventProbe;
 
+    @Mock private AuditService auditService;
+
     private SessionHandler sessionHandler;
 
     @BeforeEach
```

Comment:
I find this really nice

```
@InjectMocks private SessionHandler sessionHandler
```

This would allow you to remove the `setup` method entirely as long as everything you are injecting in the constructor is a mock which should be the case